framework/generated/generated_vulkan_state_table.h

```diff
@@ -313,6 +313,375 @@ class VulkanStateTable : VulkanStateTableBase
     std::map<format::HandleId, VideoSessionParametersKHRWrapper*> videoSessionParametersKHR_map_;
```

Comment:
This is a very superficial comment and I don't understand the architecture here, but are we maintaining these original maps and the unordered ones below for every handle live in the system at any one time?

E.g. every image created will have its handle tracked in `std::map<format::HandleId, ImageWrapper*> image_map_` and also in `std::unordered_map<VkImage, ImageWrapper*> image_map_;` below at the same time for the lifetime of the image?

Reply:
Okay, answering myself: The old set of `map`s live only as long as the system is tracking state to allow trimmed captures (deferred capture / capture from frame >1), but the new maps are static data that lives for the lifetime of the capture layer.

Reply:
The lifetime of data in both maps are almost the same. They are inserted and removed at the same Api calls, but different place. `maps` are at `EndCreateApiCallCapture` and `EndDestroyApiCallCapture`. `unordered_maps`  are at `CreateWrappedHandles` and `DestroyWrappedHandle`. I attempted to keep only one maps, but it seems not to possible,